data/json/furniture_and_terrain/appliances.json

```
@@ -225,6 +225,25 @@
     "item": "kiln",
     "copy-from": "veh_kiln"
   },
+  {
+    "type": "vehicle_part",
+    "id": "ap_chain_hoist",
+    "name": { "str": "chain hoist" },
+    "looks_like": "f_machinery_light",
+    "symbol": "7",
+    "color": "yellow_red",
+    "categories": [ "utility" ],
+    "location": "center",
+    "description": "Manual lifting equipment featuring a chain and hook.  It could be used to lift heavy loads up to 1.5 metric ton.",
```

I'm not sure, but shouldn't it be `1.5 metric tons`?
```suggestion
    "description": "Manual lifting equipment featuring a chain and hook.  It could be used to lift heavy loads up to 1.5 metric tons.",
```